docs/reference/content/builders/filters.md

```diff
@@ -150,7 +150,21 @@ This example assumes a collection that has a text index in the field `abstract`.
 containing the term `coffee`:
 
 ```java
-text("abstract", "coffee")
+text("coffee")
+```
+
+Available with MongoDB 3.2 the version 3 of the text index allows case sensitive searches, this example selects documents that have an 
```

Comment:
"Available with MongoDB 3.2,  a version 3 text index allows case-sensitive searches.  This example selects documents ..."

Below as well.  

I guess there's nothing to link to yet in the docs?